qiskit/opflow/converters/z2_taper.py

```diff
@@ -44,15 +51,12 @@ def __init__(self, num_particles: Union[int, List[int], Tuple[int, int]]):
         par_1 = 1 if (num_alpha + num_beta) % 2 == 0 else -1
         par_2 = 1 if num_alpha % 2 == 0 else -1
         self._tapering_values = [par_2, par_1]
+        self.z2_symmetries = None
```

Comment:
Why not `self._z2_symmetries`?

Reply:
I want to make this public like https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/circuit/quantumcircuit.py#L172.

Reply:
When users set z2_symmetry manually, don't you need to check the validity? In case of circuit name, it does not affect the circuit itself.

Reply:
I am not sure. Neither may not be needed like my first commit.

Reply:
If there is no use case to overwrite `Z2Taper.z2_symmetry`, it's good to make it read-only. What do you think @pbark?

Reply:
I think this property is unnecesary because this was originally `Z2Symmetry.two_qubit_reduction()` which cannot get and set Z2Symmetry. Sorry for adding and confusing.

Reply:
You can mark this discussion as resolved.